tests/op/test_run_nvnmd_train.py

```
@@ -0,0 +1,796 @@
+import itertools
```

Ruff (0.11.9)

`itertools` imported but unused

Remove unused import: `itertools`

(F401)